src/main/java/com/exactpro/th2/inframgr/DescriptorController.java

```diff
@@ -51,7 +52,7 @@ public class DescriptorController {
     private static final String UNKNOWN_ERROR = "UNKNOWN_ERROR";
 
     @Autowired
-    private Config config;
+    private KubernetesController kubernetesController;
```

Comment:
Just a small note: the controller in spring is usually responsible for handling requests from the user. If it is only used internally usually those components are called services, providers, etc.
If you want to keep the current name I don't mind. But from my point of view the naming should be changed